conda_libmamba_solver/shards.py

```diff
@@ -0,0 +1,341 @@
+# Copyright (C) 2022 Anaconda, Inc
+# Copyright (C) 2023 conda
```

Comment:
Is "conda" an entity that can own a copyright?

Reply:
🤷